### DISCUSSION TITLE
Missing Stroke fill for basic shapes

### DISCUSSION THREAD

**Discussion by mariusa** [excalidraw/excalidraw#6083] [Q&A]
Hi, 

I'm trying the online editor at https://excalidraw.com/

When importing from Libraries > Stick people, and clicking on the head, the toolbar does show "Fill"
![image](https://user-images.githubusercontent.com/1283842/211198403-cef05305-c311-4350-8123-fe0eb4d0f915.png)

After drawing & selecting a rectangle or ellipse, there's no "Fill" property in the left-side toolbar though. Curious why?

**Comment by Rostech** [comment]
Hi, you need to select a background color first. Then the fill option will pop up. 
It's hidden when the background is transparent.

**Comment by erikdemarco** [comment]
Excalidraw has many weird UX choices. Like this one for example
